src/views/JobDetails.vue

```diff
@@ -36,5 +37,57 @@ export default defineComponent({
     IonToolbar,
     JobConfiguration
   },
+  data() {
+    return {
+      title: '' as any,
+      type: '' as any,
+      freqType: '' as any,
+      jobFrequencyType: JSON.parse(process.env?.VUE_APP_JOB_FREQUENCY_TYPE as string) as any,
+      jobTitles: {
+        ...JSON.parse(process.env?.VUE_APP_JOB_TITLES as string) as any
+      },
+      jobEnums: {
+        ...JSON.parse(process.env?.VUE_APP_ODR_JOB_ENUMS as string) as any,
+        ...JSON.parse(process.env?.VUE_APP_PRODR_JOB_ENUMS as string) as any,
+        ...JSON.parse(process.env?.VUE_APP_PRD_JOB_ENUMS as string) as any,
+        ...JSON.parse(process.env?.VUE_APP_INV_JOB_ENUMS as string) as any,
+        ...JSON.parse(process.env?.VUE_APP_INITIAL_JOB_ENUMS as string) as any,
+      }
+    }
+  },
+  computed:{
+    ...mapGetters({
+      currentJob: 'job/getCurrentJob',
+      getEnumName: 'job/getEnumName'
+    }),
+  },
+  methods: {
+    viewJobConfiguration(job: any) {
+      const jobCategory = this.$route.params.category;
+      if(jobCategory !== 'pipeline') {
+        this.title = this.jobTitles[job?.systemJobEnumId];
+        const id = Object.entries(this.jobEnums).find((enums) => enums[1] == job.systemJobEnumId) as any
+        this.freqType = id && this.jobFrequencyType[id];
+      } else {
+        this.title = this.getEnumName(job.systemJobEnumId);
+        const id = Object.entries(this.jobEnums).find((enums) => enums[1] == job.systemJobEnumId) as any
+        this.freqType = id && (Object.entries(this.jobFrequencyType).find((freq) => freq[0] == id[0]) as any)[1];
+      }
+    }
+  },
+  mounted() {
```

Comment:
I think we should check if this is the right lifecycle hook, we should not need any watcher in JobConfiguration component